$NetBSD: patch-Makefile,v 1.2 2020/07/22 10:25:24 micha Exp $

Use PREFIX and PKGMANDIR from pkgsrc.
Honor CXXFLAGS and LDFLAGS from pkgsrc.
Don't set root owner on install.
Remove nonportable strip option "--strip-unneeded".
POSIX strip doesn't support options:
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/strip.html

--- Makefile.orig	2018-06-19 05:23:16.000000000 +0000
+++ Makefile
@@ -8,8 +8,8 @@
 PROGRAM=ajbsp
 
 # prefix choices: /usr  /usr/local  /opt
-PREFIX=/usr/local
-MANDIR=$(PREFIX)/share/man
+#PREFIX=/usr/local
+MANDIR=$(PREFIX)/$(PKGMANDIR)
 
 OBJ_DIR=obj_linux
 
@@ -18,16 +18,18 @@ OBJ_DIR=obj_linux
 
 WARNINGS=-Wall -Wextra -Wshadow -Wno-unused-parameter
 OPTIMISE=-O2 -fno-strict-aliasing -fwrapv -fno-exceptions -fno-rtti
-STRIP_FLAGS=--strip-unneeded
+# pkgsrc: "--strip-unneeded" is not supported on all pkgsrc architectures
+#STRIP_FLAGS= --strip-unneeded
+STRIP_FLAGS=
 
 
 #--- Internal stuff from here -----------------------------------
 
 MAN_PAGE=$(PROGRAM).6
 
-CXXFLAGS=$(OPTIMISE) $(WARNINGS) -D_THREAD_SAFE -D_REENTRANT
+#CXXFLAGS=$(OPTIMISE) $(WARNINGS) -D_THREAD_SAFE -D_REENTRANT
 
-LDFLAGS=
+#LDFLAGS=
 # LDFLAGS=-static
 
 # I needed this when using -m32 and -static:
@@ -77,9 +79,9 @@ stripped: all
 
 install: stripped
 	install -d -m 755 $(DESTDIR)$(PREFIX)/bin
-	install -o root -m 755 $(PROGRAM) $(DESTDIR)$(PREFIX)/bin/
+	install -m 755 $(PROGRAM) $(DESTDIR)$(PREFIX)/bin/
 	install -d -m 755 $(DESTDIR)$(MANDIR)/man6
-	install -o root -m 644 doc/$(MAN_PAGE) $(DESTDIR)$(MANDIR)/man6/
+	install -m 644 doc/$(MAN_PAGE) $(DESTDIR)$(MANDIR)/man6/
 
 uninstall:
 	rm -f -v $(DESTDIR)$(PREFIX)/bin/$(PROGRAM)
